compiler/haskell-ide-core/src/Development/IDE/Types/Diagnostics.hs

```diff
@@ -3,12 +3,14 @@
 
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE BlockArguments #-}
```

Comment:
We tend not to use this elsewhere. I suggest we don't start unless it offers big benefits.